sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -342,6 +345,8 @@ def query_items_change_feed(
             feed_options["partitionKey"] = self._set_partition_key(partition_key)
         if is_start_from_beginning is not None:
             feed_options["isStartFromBeginning"] = is_start_from_beginning
+        if start_time is not None and is_start_from_beginning is False and isinstance(start_time, datetime):
```

Comment:
This check isn't correct - we should not be doing an `isinstance` check for `start_time` - because this will fail silently if they pass in something else. We should just let it fail if they pass in the wrong type.